# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/CoalescedTouchData.h
# Commit: 37f6acd499b1
# Full Hash: 37f6acd499b11386a9c97f5783c89c4908692029
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2025-05-16 16:19:07
# Regressor Bug: 1550462
# File Overlap Count: 1
# Description:
#   Bug 1550462 - part 2: Make PresShell::HandleEvent dispatch preceding pointerrawupdate event r=smaug,dom-core,edgar
#   
#   This patch tries to dispatch ePointerRawUpdate with
#   PresShell::EventHandler::DispatchPrecedingPointerEvent as same as usual
#   pointer events.
# ==============================================================================

diff -r 32ba02fa32ec -r 37f6acd499b1 dom/ipc/CoalescedTouchData.h
--- a/dom/ipc/CoalescedTouchData.h	Fri May 16 09:49:43 2025 +0000
+++ b/dom/ipc/CoalescedTouchData.h	Fri May 16 09:49:44 2025 +0000
@@ -27,6 +27,8 @@
 
   nsEventStatus GetApzResponse() { return mApzResponse; }
 
+  void NotifyTouchRawUpdateOfHandled(const WidgetTouchEvent& aEvent);
+
  private:
   Touch* GetTouch(int32_t aIdentifier);
 